eo-maven-plugin/README.md

```diff
@@ -108,4 +109,17 @@ for most popular and important objects that any of you will need in order
 to write even a simple EO program. There are objects like `string`, `int`, `sprintf`,
 `stdout`, and so on. By the way, you may want to contribute there by creating new objects.
 
+## Bypassing object caching
+
+During compilation the plugin will download required objects from
+[Objectionary home](https://github.com/objectionary/home) and cache them. 
+
+Caching may be
```

Comment:
@mximp this doesn't seem accurate. This option (full name is `--update-snapshots`) forces Maven to download latest versions of snapshot dependencies from Internet _and_ cache in `~/.m2`. Thus, caching is not disable, but its update is enforced. In our case, we should do the same: keep the cache in `~/.eo` but make sure that even if an `.eo` file exists locally, we still download it _and_ cache.

Reply:
@yegor256 True - corrected